src/Razor/Razor.Design/test/testassets/ClassLibrary/ClassLibrary.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <_RazorMSBuildRoot>$(SolutionRoot)Razor.Design\src\bin\$(Configuration)\netstandard2.0\</_RazorMSBuildRoot>
+    <IsPackable>false</IsPackable>
+    <IsImplementationProject>false</IsImplementationProject>
+    <IsProjectReferenceProvider>true</IsProjectReferenceProvider>
```

Comment:
This doesn't seem to be respected for this project. A few tests are failing because of it.

Reply:
Try just using a regular ProjectReference for now. "ClassLibrary" is a very generic name for an asset. It might be conflicting with other projects or packages named 'ClassLibrary'.